tests/model_hub_tests/hf_hub_tests/models/nightly

```diff
@@ -0,0 +1,54 @@
+hf-tiny-model-private/tiny-random-GPTJForCausalLM
```

Comment:
is it PyTorch model, right? Please put all of these files to Pytorch sub-directory